MyEtherWallet/MyEtherWallet
# Handle Ledger Error with Non-U2F Browser

Before, if your browser didn't support U2F, and you tried it unlock Ledger, it was trying to render an `Error` object, which doesn't work. Now it turns an error object into a string.

### Steps to Test

* Open up Firefox
* Go to /account
* Unlock Ledger
* See "Error: U2F not supported"
